framework/include/vectorpostprocessors/VectorPostprocessor.h

```diff
@@ -123,3 +123,56 @@ class VectorPostprocessorContext : public ReporterGeneralContext<T>
   ScatterVectorPostprocessorValue _scatter_value;
   ScatterVectorPostprocessorValue _scatter_value_old;
 };
+
```

Comment:
Did you try just moving it above the explicit instantiation?

Reply:
Tried and works, going with that approach.